doc/modules/changes/20210709_sliu

```diff
@@ -0,0 +1,3 @@
+Fixed: The "prescribed_dilation" and "additional Stokes RHS" options now work correctly with the Newton solver in newton.stokes.cc.
```

Comment:
```suggestion
Fixed: The "prescribed_dilation" and "additional Stokes RHS" options now work correctly with the Newton solver.
```

Looks good otherwise! Thank you.

Reply:
Fixed.

Also updated the test outputs